ci/prow/config_knative.yaml

```diff
@@ -172,6 +172,10 @@ presubmits:
       dot-dev: true
     - go-coverage: true
       dot-dev: true
+    - custom-test: integration-tests-on-latest-serving
```

Comment:
We do not need this custom job for PR build so far, only the one in the periodic job is needed. Thx.